client/src/containers/CampaignAdmin/CampaignAdminItem.js

```diff
@@ -1,12 +1,66 @@
 import React from 'react';
 import SilverStripeComponent from 'lib/SilverStripeComponent';
 import i18n from 'i18n';
+import { Tooltip, OverlayTrigger } from 'react-bootstrap-ss';
+import formatWrittenNumber from 'lib/formatWrittenNumber';
 
 /**
  * Describes an individual campaign item
  */
 class CampaignAdminItem extends SilverStripeComponent {
 
+  /**
+   * @return integer
+   */
+  getNumReferTo() {
+    const numReferTo = (
+      this.props.item._links &&
+      this.props.item._links.refer_to &&
+      this.props.item._links.refer_to.length
+    );
+
+    return numReferTo || 0;
+  }
+
+  /**
+   * @return integer
+   */
+  getNumReferredBy() {
+    const numReferredBy = (
+      this.props.item._links &&
+      this.props.item._links.referenced_by &&
+      this.props.item._links.referenced_by.length
+    );
+
+    return numReferredBy || 0;
+  }
+
+  /**
+   * @return string
+   */
+  getReferToTooltipText() {
+    const numReferTo = this.getNumReferTo();
+
+    const text = numReferTo < 2 ?
+      i18n._t('CampaignAdmin.LINKED_TO_SINGULAR', 'Links to %s item') :
+      i18n._t('CampaignAdmin.LINKED_TO_PLURAL', 'Links to %s items');
```

Comment:
You're right, probably isn't a story for the javascript side (there's one for the php side which was done), I'll be happy to raise one :)
but yeah, I don't like the hardcoding for pluralisation, it only caters for english really.